### PR TITLE
Create migration db

### PR DESCRIPTION
**Pull request by OlegNovatskiy**
In it pull request are commits for create sql files for migration db with flyway. And was create the config file for hibernate.

### CODE REVIEW

File: src/main/resources/db/migration/V1_1__Create_table_for_db.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+	`title` varchar(50) NOT NULL,
+	PRIMARY KEY (`id`)
+);
+
+CREATE TABLE `modelscars` (
+	`id` int(11) NOT NULL AUTO_INCREMENT,
+	`id_brand` int(11) NOT NULL,
+	`title` varchar(50) NOT NULL,
+	PRIMARY KEY (`id`)
+);
```
Comment (ZVIproject) [inline src/main/resources/db/migration/V1_1__Create_table_for_db.sql]: add to all tables columns date time and time stamp

Comment (OlegNovatskiy) [inline src/main/resources/db/migration/V1_1__Create_table_for_db.sql]: it's not required in there

Comment (OlegNovatskiy) [inline src/main/resources/db/migration/V1_1__Create_table_for_db.sql]: DONE

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+flyway.schemas = Kerpach_AIS
+flyway.locations=classpath:db/migration
+flyway.encoding = utf8
+flyway.sqlMigrationPrefix=V
+##Max version of database
+flyway.target = 1.2
+#Validate migration to db (default - true)
+#flyway.validateOnMigrate = true
+###----- Don't forgot change it property -----###
+flyway.cleanOnValidationError = true
```
Comment (af150416) [inline src/main/resources/application.properties]: do you really  need all this flyway properties?
may be in your case enough 
flyway.sqlMigrationPrefix=V
flyway.out-of-order=true

Comment (OlegNovatskiy) [inline src/main/resources/application.properties]: It's properties use in during development

**Pull request merged**